test/menus/menu_path_test.dart

```diff
@@ -0,0 +1,123 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:overlord/overlord.dart';
+
+void main() {
+  group("Menus > menu paths >", () {
+    group("empty path > ", () {
+      test("is never a sub-path", () {
+        expect(
+          const MenuPath([]).isSubPathOf(
+            const MenuPath(["file", "new", "document"]),
+          ),
+          isFalse,
+        );
+
+        expect(
+          const MenuPath([]).isSubPathOf(
+            const MenuPath([]),
+          ),
+          isFalse,
+        );
+      });
+
+      test("is never a container path", () {
+        expect(
+          const MenuPath(["file", "new", "document"]).containsPath(
+            const MenuPath([]),
+          ),
+          isFalse,
+        );
```

Comment:
Shouldn't this be inverted?  The test reads "empty path > is never a container path", but here the container is `["file", "new", "document"]`, right?